editor/src/clj/editor/properties_view.clj

```diff
@@ -663,7 +663,6 @@
                     (ui/add-styles! ["clear-button" "button-small"])
                     (ui/on-action! (fn [_]
                                      (properties/clear-override! (property-fn key))
-                                     (ui/suppress-auto-commit! control)
```

Comment:
I found why it was added https://github.com/defold/defold/commit/fbe47b48787963a396ebb718a0b04a3e4e1c9bd8
but I can't reproduce anything like that as well as I see that commit (`update-fn`) never called after reset even without this line. 
Maybe it was changed since then.